packages/server/src/events/tasks.ts

```diff
@@ -25,7 +25,51 @@ export async function CreateArticleStory(_email: string, articleId: number) {
  * Task ( Runs after the article's story is created successfully ).
  * Distribute the article story among the followers of the article's author
  */
-export async function DistributeArticleStory(_email: string, _articleId: number) {}
+export async function DistributeArticleStory(_email: string, _articleId: number) {
+    const prisma = PrismaClientSingleton.prisma;
+    const article = await prisma.article.findUnique({
+        where: {
+            id: _articleId,
+        },
+    });
+
+    const authorId = article?.userId!;
+
+    const followers = await prisma.user.findUnique({
+        where: {
+            id: authorId,
+        },
+        select: {
+            followers: {
+                select: {
+                    followerId: true,
+                },
+            },
+        },
+    });
+
+    const followerIds = followers?.followers.map((follower) => follower.followerId);
+
+    const articleStory = await prisma.articleStory.findUnique({
+        where: {
+            articleId: _articleId
+        },
+    });
+
+    const expirationTime = add(new Date(), { days: 1 }); // Calculate expiration time (1 day from now)
```

Comment:
It should be dependent on the Article's published time but it is alright it will work for now.